ocs_ci/ocs/ui/page_objects/page_navigator.py

```diff
@@ -100,6 +100,15 @@ def nav_odf_default_page(self):
         logger.info(f"Default page is {self.driver.title}")
         return default_tab
 
+    def nav_object_storage_page(self):
+        """
+        Navigate to Object Storage page
+
+        """
+        self.choose_expanded_mode(mode=True, locator=self.page_nav["Storage"])
+        self.do_click(locator=self.page_nav["object_storage_page"], timeout=90)
+        self.page_has_loaded(retries=15)
```

Comment:
it would be perfect here to create object_storage.py at ocs_ci/ocs/ui/page_objects, ObjectStorage class and return it here. Same as others in page_objects directory.

This way we can store all related functionality there.